ui/raidboss/data/04-sb/trial/lakshmi-ex.txt

```diff
@@ -11,96 +11,99 @@ hideall "Alluring Arm"
-154.5 "Alluring Arm" # end of castbar
-157.0 "Hands Of Grace/Beauty" duration 11.5 # hat -> disappearing
-161.5 "The Pull Of Light" # end of castbar
-171.7 "Stotram" # end of castbar
-179.0 "The Pall Of Light" # end of castbar
-188.2 "The Pull Of Light" # end of castbar
+107.2 "Hand Of Grace" sync /:Lakshmi:214C:/ window 110,0
+114.4 "The Pull Of Light" sync /:Lakshmi:215E:/
+121.4 "Blissful Arrow" sync /:Lakshmi:214F:/
+122.0 "Blissful Spear" sync /:Lakshmi:2151:/
+124.6 "Stotram" sync /:Lakshmi:2147:/
+135.8 "The Pull Of Light" sync /:Lakshmi:215E:/
+143.1 "The Path Of Light" sync /:Lakshmi:215A:/
+153.2 "Alluring Arm" sync /:Lakshmi:214E:/
+160.4 "The Pull Of Light" sync /:Lakshmi:215E:/
+167.5 "Blissful Hammer" sync /:Lakshmi:21DC:/
+170.6 "Stotram" sync /:Lakshmi:2147:/
+177.8 "The Pall Of Light" sync /:Lakshmi:215C:/
+187.0 "The Pull Of Light" sync /:Lakshmi:215E:/
 
 ### Chanchala #1
-194.0 "--sync--" sync /:2148:Lakshmi/ window 20,20
-196.5 "Chanchala" # end of castbar
-210.2 "Divine Denial" # end of castbar
-221.2 "Hand Of Beauty" duration 11.5 # hat -> disappearing
-225.5 "The Pull Of Light" # end of castbar
-228.5 "The Path Of Light" duration 5.5 # hat -> cleave
-246.0 "Divine Desire" # end of castbar
-249.5 "--chanchala end--"
+195.2 "Chanchala #1" sync /:Lakshmi:2148:/
+209.0 "Divine Denial" sync /:Lakshmi:2149:/
+217.2 "Hand Of Beauty" sync /:Lakshmi:214D:/
+224.4 "The Pull Of Light" sync /:Lakshmi:215F:/
+231.4 "Blissful Hammer" sync /:Lakshmi:21DD:/
+231.7 "The Path Of Light" sync /:Lakshmi:215B:/
+245.0 "Divine Desire" sync /:Lakshmi:214B:/
+247.1 "--chanchala end--"
 
-258.2 "The Pull Of Light" # end of castbar
-266.2 "The Path Of Light" duration 5.5 # hat -> cleave
-279.0 "Stotram" # end of castbar
-282.0 "--untargetable--"
+257.3 "The Pull Of Light" sync /:Lakshmi:215E:/
+269.7 "The Path Of Light" sync /:Lakshmi:215A:/
+277.8 "Stotram" sync /:Lakshmi:2147:/
+281.5 "--untargetable--"
 
 ### More Adds
-287.5 "Adds Appear"
+287.1 "--targetable--"
 # infinite loop
 335.0 "--sync--" sync /.*/ window 0,50 jump 300
 
 ### Hugging
-400.0 "--sync--" sync /:Lakshmi:2342:Jagadishwari/ window 400,0
-421.0 "/dance"
+400.0 "Jagadishwari" sync /:Lakshmi:2342:/ window 400,0
+421.4 "/dance"
+421.4 "--sync--" sync /:Lakshmi:2159:/
+437.1 "Alluring Embrace" sync /:Lakshmi:2343:/
+441.5 "--targetable--"
 
 ### Chanchala #2
-441.5 "--sync--" sync /:2148:Lakshmi/ window 20,20
-444.2 "Chanchala" # end of castbar
-459.5 "Divine Doubt" # end of castbar
-470.5 "The Path Of Light" duration 5.5 # hat -> cleave
-485.2 "The Pull Of Light" # end of castbar
-488.5 "--chanchala end--"
+441.7 "--sync--" sync /:2148:Lakshmi/ window 50,10
+444.7 "Chanchala #2" sync /:Lakshmi:2148:/
+458.5 "Divine Doubt" sync /:Lakshmi:214A:/
+473.9 "The Path Of Light" sync /:Lakshmi:215B:/
+484.0 "The Pull Of Light" sync /:Lakshmi:215F:/
+486.0 "--chanchala end--"
+509.5 "The Pull Of Light" sync /:Lakshmi:215E:/
+511.5 "Blissful Spear (Z)" sync /:Lakshmi:2166:/
```

Comment:
Here and elsewhere, what do the parentheses mean?

Reply:
These are the patterns that the vril show up in.  In extreme, there's ground aoes (lines in a Z pattern, circle on the outside, etc) with vril in them.

Reply:
Huh, okay. I didn't really pay attention to the exact patterns going through, that's useful stuff.